tests/constraints.in

```diff
@@ -2,4 +2,4 @@
 # and antsibull-docs that production builds rely upon.
 
 sphinx == 7.2.5
-antsibull-docs == 2.11.0  # currently approved version
+antsibull-docs == 2.13.1  # currently approved version
```

Comment:
@felixfontein Hey, thanks for bumping this but I'm noticing this in the RTD build logs:

```
ERROR: Cannot install antsibull-docs~=2.7 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested antsibull-docs~=2.7
    The user requested (constraint) antsibull-docs==2.13.1
```

Logs: https://readthedocs.org/api/v2/build/25422658.txt

I can see the pin in `requirements-relaxed.in`: https://github.com/ansible/ansible-documentation/blob/a8226b1bddfc7b50a78d9d33158dac0da7848ae6/tests/requirements-relaxed.in#L15

And this is referenced in `requirements.in`: https://github.com/ansible/ansible-documentation/blob/a8226b1bddfc7b50a78d9d33158dac0da7848ae6/tests/requirements.in#L5

I'm curious to try this locally and understand it better but what do you think is going on here?

Reply:
That seems to have been a random fluke, re-triggering the RTD build magically made this go away.

Reply:
(According to https://packaging.python.org/en/latest/specifications/version-specifiers/#compatible-release `~=2.7` and `==2.13.1` are no contradiction.)

Reply:
thanks for looking into it @felixfontein